scripts/xtensa-build-all.sh

```diff
@@ -281,10 +281,10 @@ do
 		PATH=$pwd/../$HOST/bin:$OLDPATH
 		COMPILER="gcc"
 
-		if [ $j == "byt" ] || [ $j == "cht" ] || [ $j == "sue" ]
-		then
-			DEFCONFIG_PATCH="_gcc"
-		fi
+		case $j in
+			byt|cht|cnl|sue) DEFCONFIG_PATCH="_gcc"
```

Comment:
Should this be in previous patch ?

Reply:
@lgirdwood I believe the latest version is ok. The first patch just swaps gcc/xcc extensions, the second one adds cnl as an exception for gcc builds as well.